diff -ur wireshark-1.2.1/Makefile.am wireshark-1.2.1-btbb/Makefile.am
--- wireshark-1.2.1/Makefile.am	2009-07-19 16:23:54.000000000 -0600
+++ wireshark-1.2.1-btbb/Makefile.am	2009-08-19 10:28:14.000000000 -0600
@@ -244,6 +244,7 @@
 -include plugins/Custom.make
 plugin_ldadd = $(_CUSTOM_plugin_ldadd_) \
 	-dlopen plugins/asn1/asn1.la \
+	-dlopen plugins/btbb/btbb.la \
 	-dlopen plugins/docsis/docsis.la \
 	-dlopen plugins/ethercat/ethercat.la \
 	-dlopen plugins/giop/cosnaming.la \
diff -ur wireshark-1.2.1/configure.in wireshark-1.2.1-btbb/configure.in
--- wireshark-1.2.1/configure.in	2009-07-19 16:23:54.000000000 -0600
+++ wireshark-1.2.1-btbb/configure.in	2009-08-19 10:28:14.000000000 -0600
@@ -1703,6 +1703,7 @@
   packaging/svr4/pkginfo
   plugins/Makefile
   plugins/asn1/Makefile
+  plugins/btbb/Makefile
   plugins/docsis/Makefile
   plugins/ethercat/Makefile
   plugins/giop/Makefile
Only in wireshark-1.2.1-btbb: configure.in.orig
diff -ur wireshark-1.2.1/packaging/nsis/Makefile.nmake wireshark-1.2.1-btbb/packaging/nsis/Makefile.nmake
--- wireshark-1.2.1/packaging/nsis/Makefile.nmake	2009-07-19 16:23:47.000000000 -0600
+++ wireshark-1.2.1-btbb/packaging/nsis/Makefile.nmake	2009-08-19 10:28:14.000000000 -0600
@@ -44,6 +44,7 @@
 
 PLUGINS= \
 	../../plugins/asn1/asn1.dll \
+	../../plugins/btbb/btbb.dll \
 	../../plugins/docsis/docsis.dll \
 	../../plugins/ethercat/ethercat.dll \
 	../../plugins/giop/coseventcomm.dll \
diff -ur wireshark-1.2.1/packaging/nsis/wireshark.nsi wireshark-1.2.1-btbb/packaging/nsis/wireshark.nsi
--- wireshark-1.2.1/packaging/nsis/wireshark.nsi	2009-07-19 16:23:47.000000000 -0600
+++ wireshark-1.2.1-btbb/packaging/nsis/wireshark.nsi	2009-08-19 10:28:14.000000000 -0600
@@ -809,6 +809,7 @@
 ;-------------------------------------------
 SetOutPath '$INSTDIR\plugins\${VERSION}'
 File "..\..\plugins\asn1\asn1.dll"
+File "..\..\plugins\btbb\btbb.dll"
 File "..\..\plugins\docsis\docsis.dll"
 File "..\..\plugins\ethercat\ethercat.dll"
 File "..\..\plugins\giop\coseventcomm.dll"
diff -ur wireshark-1.2.1/plugins/Makefile.am wireshark-1.2.1-btbb/plugins/Makefile.am
--- wireshark-1.2.1/plugins/Makefile.am	2009-07-19 16:21:53.000000000 -0600
+++ wireshark-1.2.1-btbb/plugins/Makefile.am	2009-08-19 10:28:14.000000000 -0600
@@ -24,6 +24,7 @@
 -include Custom.make
 SUBDIRS = $(_CUSTOM_SUBDIRS_) \
 	asn1 \
+	btbb \
 	docsis \
 	ethercat \
 	giop \
diff -ur wireshark-1.2.1/plugins/Makefile.nmake wireshark-1.2.1-btbb/plugins/Makefile.nmake
--- wireshark-1.2.1/plugins/Makefile.nmake	2009-07-19 16:21:53.000000000 -0600
+++ wireshark-1.2.1-btbb/plugins/Makefile.nmake	2009-08-19 10:28:14.000000000 -0600
@@ -33,6 +33,9 @@
 	cd asn1
 	$(MAKE) /$(MAKEFLAGS) -f Makefile.nmake $(PLUGIN_TARGET)
 	cd ..
+	cd btbb
+	$(MAKE) /$(MAKEFLAGS) -f Makefile.nmake $(PLUGIN_TARGET)
+	cd ..
 	cd docsis
 	$(MAKE) /$(MAKEFLAGS) -f Makefile.nmake $(PLUGIN_TARGET)
 	cd ..
@@ -86,6 +89,7 @@
 !IFDEF ENABLE_LIBWIRESHARK
 	cd..
 	xcopy plugins\asn1\*.dll $(INSTALL_DIR)\plugins\$(VERSION) /d
+	xcopy plugins\btbb\*.dll $(INSTALL_DIR)\plugins\$(VERSION) /d
 	xcopy plugins\docsis\*.dll $(INSTALL_DIR)\plugins\$(VERSION) /d
 	xcopy plugins\ethercat\*.dll $(INSTALL_DIR)\plugins\$(VERSION) /d
 	xcopy plugins\giop\*.dll $(INSTALL_DIR)\plugins\$(VERSION) /d
