diff -rux entries wireshark-svn/configure.in wireshark/configure.in
--- wireshark-svn/configure.in	2009-08-11 13:06:33.000000000 -0400
+++ wireshark/configure.in	2009-08-11 13:02:37.000000000 -0400
@@ -1739,6 +1739,7 @@
   packaging/svr4/pkginfo
   plugins/Makefile
   plugins/asn1/Makefile
+  plugins/btbb/Makefile
   plugins/docsis/Makefile
   plugins/ethercat/Makefile
   plugins/giop/Makefile
Only in wireshark: configure.in.orig
diff -rux entries wireshark-svn/Makefile.am wireshark/Makefile.am
--- wireshark-svn/Makefile.am	2009-08-11 13:06:33.000000000 -0400
+++ wireshark/Makefile.am	2009-08-11 13:02:37.000000000 -0400
@@ -264,6 +264,7 @@
 -include plugins/Custom.make
 plugin_ldadd = $(_CUSTOM_plugin_ldadd_) \
 	-dlopen plugins/asn1/asn1.la \
+	-dlopen plugins/btbb/btbb.la \
 	-dlopen plugins/docsis/docsis.la \
 	-dlopen plugins/ethercat/ethercat.la \
 	-dlopen plugins/giop/cosnaming.la \
Only in wireshark: Makefile.am.orig
diff -rux entries wireshark-svn/packaging/nsis/Makefile.nmake wireshark/packaging/nsis/Makefile.nmake
--- wireshark-svn/packaging/nsis/Makefile.nmake	2009-08-11 13:06:32.000000000 -0400
+++ wireshark/packaging/nsis/Makefile.nmake	2009-08-11 13:02:37.000000000 -0400
@@ -44,6 +44,7 @@
 
 PLUGINS= \
 	../../plugins/asn1/asn1.dll \
+	../../plugins/btbb/btbb.dll \
 	../../plugins/docsis/docsis.dll \
 	../../plugins/ethercat/ethercat.dll \
 	../../plugins/giop/coseventcomm.dll \
diff -rux entries wireshark-svn/packaging/nsis/wireshark.nsi wireshark/packaging/nsis/wireshark.nsi
--- wireshark-svn/packaging/nsis/wireshark.nsi	2009-08-11 13:06:32.000000000 -0400
+++ wireshark/packaging/nsis/wireshark.nsi	2009-08-11 13:02:37.000000000 -0400
@@ -874,6 +874,7 @@
 ;-------------------------------------------
 SetOutPath '$INSTDIR\plugins\${VERSION}'
 File "..\..\plugins\asn1\asn1.dll"
+File "..\..\plugins\btbb\btbb.dll"
 File "..\..\plugins\docsis\docsis.dll"
 File "..\..\plugins\ethercat\ethercat.dll"
 File "..\..\plugins\giop\coseventcomm.dll"
Only in wireshark/packaging/nsis: wireshark.nsi.orig
Only in wireshark/plugins: btbb
diff -rux entries wireshark-svn/plugins/Makefile.am wireshark/plugins/Makefile.am
--- wireshark-svn/plugins/Makefile.am	2009-08-11 13:06:01.000000000 -0400
+++ wireshark/plugins/Makefile.am	2009-08-11 13:02:37.000000000 -0400
@@ -24,6 +24,7 @@
 -include Custom.make
 SUBDIRS = $(_CUSTOM_SUBDIRS_) \
 	asn1 \
+	btbb \
 	docsis \
 	ethercat \
 	giop \
diff -rux entries wireshark-svn/plugins/Makefile.nmake wireshark/plugins/Makefile.nmake
--- wireshark-svn/plugins/Makefile.nmake	2009-08-11 13:06:01.000000000 -0400
+++ wireshark/plugins/Makefile.nmake	2009-08-11 13:02:37.000000000 -0400
@@ -33,6 +33,9 @@
 	cd asn1
 	$(MAKE) /$(MAKEFLAGS) -f Makefile.nmake $(PLUGIN_TARGET)
 	cd ..
+	cd btbb
+	$(MAKE) /$(MAKEFLAGS) -f Makefile.nmake $(PLUGIN_TARGET)
+	cd ..
 	cd docsis
 	$(MAKE) /$(MAKEFLAGS) -f Makefile.nmake $(PLUGIN_TARGET)
 	cd ..
@@ -86,6 +89,7 @@
 !IFDEF ENABLE_LIBWIRESHARK
 	cd..
 	xcopy plugins\asn1\*.dll $(INSTALL_DIR)\plugins\$(VERSION) /d
+	xcopy plugins\btbb\*.dll $(INSTALL_DIR)\plugins\$(VERSION) /d
 	xcopy plugins\docsis\*.dll $(INSTALL_DIR)\plugins\$(VERSION) /d
 	xcopy plugins\ethercat\*.dll $(INSTALL_DIR)\plugins\$(VERSION) /d
 	xcopy plugins\giop\*.dll $(INSTALL_DIR)\plugins\$(VERSION) /d
